.travis.yml

```diff
@@ -1,24 +1,27 @@
 language:
-- python
+  - python
 sudo: false
 python:
-- '2.7'
-- '3.3'
-- '3.4'
-- '3.5'
+  - '2.7'
+  - '3.3'
+  - '3.4'
+  - '3.5'
 env:
   - INSTALLATION="python setup.py develop" XDIST=1
   # unfortunatelly, xdist hangs if there's any error in the slave threads, and there, pytest fires all the hooks
   - INSTALLATION="pip install ." XDIST=0
+before_install:
+  - mkdir /tmp/dynamodb # create the folder for dynamodb package
+  - wget -O - http://dynamodb-local.s3-website-us-west-2.amazonaws.com/dynamodb_local_latest | tar xz --directory /tmp/dynamodb # download the package and unpack it
```

Comment:
We are going to use the link with `latest` in version name or should we use the one  version defined i.e. `2016-05-17`: http://dynamodb-local.s3-website-us-west-2.amazonaws.com/dynamodb_local_2016-05-17.tar.gz ? We could add some sha checking trough. But I was looking for page with given the sha versions and I couldn't find it.